src/components/simulation/ConsumptionEmailResult.vue

```diff
@@ -101,8 +99,10 @@ const openPrivacy = () => {
           <span
             @click="openPrivacy()"
             class="font-medium underline decoration-1 hover:cursor-pointer"
-            >Politique de confidentialité</span
+            data-testid="privacy-link"
```

Comment:
What is the difference between id (l. 75) and data-testid ?

Reply:
data-testid is an id made only for test. Id is the generic id of element